hogvm/__tests__/__snapshots__/dateFormat.stdout

```diff
@@ -0,0 +1,43 @@
+2009-02-11 06:42:23
+2009-02-11 07:42:23
+2009-02-11 01:42:23
+-----
+%a: Wed
+%b: Feb
+%c: 02
+%C: 09
+%d: 11
+%D: 02/11/09
+%e: 11
+%F: 2009-02-11
+%g: 09
+%G: 2009
+%h: 06
+%H: 06
```

Comment:
Nit: Either this doesn't work or your test data doesn't prove that it does 😉

Reply:
Indeed... 😅, added 43 thousand seconds to the timestamp to fix this.